### ISSUE TITLE
No label for the actions menu in home view

### ISSUE THREAD

**Issue by skoeva** [headlamp-k8s/headlamp#2526]
## Describe the bug
The actions menu does not have a label

## To Reproduce
Steps to reproduce the bug:
1. Open Headlamp
2. Hover over actions menu on a cluster
3. Notice that it has no label

## Are you able to fix this issue?
Yes

## Additional Context
The issue is shown here (hovering over the ... on the right):

![image](https://github.com/user-attachments/assets/b6d6d6fe-4374-48a5-94ce-0f80966a30c1)
